src/components/CenturiesFiltering/CenturiesFilltering.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+
+import { SearchLink } from '../../SearchLink';
+
+export const CenturiesFilltering = () => {
+  const [searchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries') || [];
+  const centuriesArr = ['16', '17', '18', '19', '20'];
+
+  return (
+    <div className="panel-block">
+      <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
+        <div className="level-left">
+          {centuriesArr.map((centurie) => (
+            <SearchLink
+              data-cy="century"
+              className={classNames(
+                'button',
+                'mr-1',
+                { 'is-info': centuries.includes(centurie) },
+              )}
+              params={{
+                centuries: centuries.includes(centurie)
+                  ? centuries.filter((c) => c !== centurie)
+                  : [...centuries, centurie],
+              }}
```

Comment:
move it to the separate variable

Reply:
comment wasn't fixed

Reply:
![image](https://user-images.githubusercontent.com/122989585/230763630-ca2c394d-17bb-4259-9847-bd277b0fe06e.png)